mappings/net/minecraft/server/chat/ChatFilterer.mapping

```diff
@@ -0,0 +1,59 @@
+CLASS net/minecraft/class_5514 net/minecraft/server/chat/ChatFilterer
```

Comment:
InputFilter? ContentFilter? ContentFilterClient?

Reply:
That's my current problem right now, this isn't strictly for chat. It's used by anything where the user and type something, like renaming an item or editing a sign

Reply:
It has been renamed to `TextFilterer`. Should I drop the `er`?

Reply:
Perhaps `TextFilterManager`?

Reply:
I would like more feedback on this

Reply:
I haven't received much feedback on this, marking as resolved